src/50_Samples_%26_Templates/Poll.html

```diff
@@ -0,0 +1,154 @@
+
+<style amp-custom>
+  #poll1 li {
+            position: relative;
+            padding: 1em 0;
+            list-style: none;
+            margin: 0;
+            display: block;
+            height: 20px;
+        }
+        #poll1.amp-form-submit-success fieldset {
+            display: none;
+        }
+        .percentage-container {
+            font-size: 0;
+            display: flex;
+        }
+        .percentage-container-fixed {
+          height: 100%;
+          position: absolute;
+          font-size: 0;
+          right: 80px;
+          left: 160px;
+        }
+        .one-pc-fixed {
+           width: 1%;
+           display: inline-block;
+           background: #eb407a;
+           height: 100%;
+       }
+        #header1 {
+          padding-left: 32px;
+        }
+        form.amp-form-submit-success [submit-success] {
+        flex: 0 0 100%;
+      }
+      .fixed-size-cell {
+        width: 90px;
+        display: inline-block;
+        max-width: 90px;
+      }
+      .votes-cell {
+        float: right;
+        width: 60px;
+      }
+      .table-row {
+        position: relative;
+        margin-top: 16px;
+      }
+      .table {
+        margin: 16px;
+      }
+
+</style>
+</head>
+<body>
+
+<h4 id="header1">Flightless bird poll</h4>
+<!-- #### Poll -->
+<!--
+We use `amp-form` to implement a poll. To avoid multiple votes by the same user, we need to identify which users already voted.
+We use a cookie called `POLL_USER_ID` to identify the user.
+Given in Safari the default behavior is to block third-party cookies from sites that have not been previously visited, this approach may not work if the page is visited via the AMP Cache. In this case we fallback to checking for duplicate entries via the `CLIENT_ID`.
+The `CLIENT_ID` is an unique identifier for the user when
+accessing a specific origin, for example the original page or the cached version of it, this means that the same user will generate two different
+values when accessing different origins. That's why we use a custom cookie to track cross origin access to the poll page.
+The `CLIENT_ID` variable can be passed via an amp-form by declaring an hidden input
+value with `default-value="CLIENT_ID(poll)`.
+Read more about variable substitution [here](https://github.com/ampproject/amphtml/blob/ab78ba87cbf0408ca0ddcfcba7e666e1c1152051/extensions/amp-form/amp-form.md#variable-substitutions). -->
+
+<form method="post" action-xhr="<%host%>/samples_templates/poll/submit" target="_blank"
+      id="poll1"
+      custom-validation-reporting="as-you-go">
+    <input name="clientId" type="hidden" value="CLIENT_ID(poll)" data-amp-replace="CLIENT_ID">
```

Comment:
Should this be `CLIENT_ID(POLL_USER_ID)` instead of `CLIENT_ID(poll)`? I don't see where the `poll` cookie is created or read.